Lib9c/Model/State/WeeklyArenaState.cs

```diff
@@ -474,6 +474,31 @@ public int UpdateV3(AvatarState avatarState, ArenaInfo enemyInfo, BattleLog.Resu
             return Score - current;
         }
 
+        public int Update(AvatarState avatarState, ArenaInfo enemyInfo, BattleLog.Result result)
+        {
+            switch (result)
+            {
+                case BattleLog.Result.Win:
+                    ArenaRecord.Win++;
+                    break;
+                case BattleLog.Result.Lose:
+                    ArenaRecord.Lose++;
+                    break;
+                case BattleLog.Result.TimeOver:
+                    ArenaRecord.Draw++;
+                    return 0;
```

Comment:
I think dailyChallengeCount doesn't update this case, is this correct?

Reply:
Oh..! Thanks to find that!